### PR TITLE
Design tdd matplotlib-applied.ipynb

### PR DESCRIPTION
**Pull request by Zhaochengxiann**
Submit designTDD matplotlib-applied.ipynb

### CODE REVIEW

File: open-machine-learning-jupyter-book/assignments/set-up-env/second-assignment.ipynb [hunk 1]
```diff
@@ -26,7 +26,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/assignments/set-up-env/second-assignment.ipynb:29]: Why you submit change for this file?

File: open-machine-learning-jupyter-book/assignments/set-up-env/first-assignment.ipynb [hunk 2]
```diff
@@ -24,22 +24,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/assignments/set-up-env/first-assignment.ipynb:27]: Why your submit this file to the PR? This does not suppose to be your change in this PR.

File: open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb [hunk 3]
```diff
@@ -0,0 +1,406 @@
+{
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb:1]: This file should be in the `assignments/data-science/` folder.

File: open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+ "cells": [
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb:2]: Please follow this to add the license section and acknowledge section. https://github.com/open-academy/machine-learning/blob/main/open-machine-learning-jupyter-book/assignments/prerequisites/python-programming-advanced.ipynb

**Comment by zhangqi444** [comment]
Please give a more meaningful PR name always.

File: open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb (hunk 3 above)
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb:1]: <p>Please clear the output of the assignment before you submit the PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

**Comment by zhangqi444** [comment]
> Please give a more meaningful PR name always.

@Zhaochengxiann please have a look at this.

File: open-machine-learning-jupyter-book/assignments/set-up-env/second-assignment.ipynb (hunk 1 above)
Comment (Zhaochengxiann) [reply open-machine-learning-jupyter-book/assignments/set-up-env/second-assignment.ipynb:29]: Sorry, this is the homework that was “ramped up“ before, I didn't have PR before, and I deleted him now

File: open-machine-learning-jupyter-book/assignments/set-up-env/first-assignment.ipynb (hunk 2 above)
Comment (Zhaochengxiann) [reply open-machine-learning-jupyter-book/assignments/set-up-env/first-assignment.ipynb:27]: Sorry, this is the homework that was “ramped up“ before, I didn't have PR before, and I deleted him now

File: open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb (hunk 3 above)
Comment (Zhaochengxiann) [reply open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb:1]: I think I have already put this file in the assignments/data-science/folder, i don't know where the problem is.

File: open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb (hunk 4 above)
Comment (Zhaochengxiann) [reply open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb:2]: done.

File: open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb (hunk 3 above)
Comment (Zhaochengxiann) [reply open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb:1]: ok

File: open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb [hunk 5]
```diff
@@ -0,0 +1,342 @@
+{
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb:1]: <p>You don't need to add an extra line for the link. You can put it inline of the first line. Please refer to how we do this in our other page.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Comment by zhangqi444** [comment]
Please not delete the files you are not going to change. You need to exclude the first-assignment and second-assignment from your commit.

**Pull request closed without merging**